test/unit/Changelog/BumpAndCommitChangelogVersionViaKeepAChangelogTest.php

```diff
@@ -225,29 +228,29 @@ private function createMockChangelog(): string
 
     private const CHANGELOG_STUB = <<< 'CHANGELOG'
         # Changelog
-        
```

Comment:
Do the tests that use these continue to work? I'd originally left these spaces in as they get trimmed when the nowdoc is evaluated by the engine, but I'm not sure what happens if they're omitted.